src/main/java/pl/wedding/management/weddinghall/WeddingHall.java

```diff
@@ -0,0 +1,41 @@
+package pl.wedding.management.weddinghall;
+
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+
+@RequestMapping("/weddinghalls")
+@RestController
+class WeddingHallController {
+
+    private final WeddingHallService weddingHallService;
+
+    WeddingHallController(WeddingHallService weddingHallService){
+        this.weddingHallService=weddingHallService;
+    }
+
+    @GetMapping
+    public ResponseEntity getAllWeddingHalls() {
+        return ResponseEntity.ok(weddingHallService.findAll());
+    }
+
+    @GetMapping("/{id}")
+    public ResponseEntity getWeddingHallById(@RequestParam("id") Long id) {
+        return ResponseEntity.ok(weddingHallService.findById(id));
```

Comment:
Zupełnie niepotrzebny strzał do serwisu - od razu możemy strzelać do repo.

Reply:
Ok, zrobione juz.

Reply:
> Zupełnie niepotrzebny strzał do serwisu - od razu możemy strzelać do repo.
@wedding-hall-adm  I don't agree here. I'm for respecting MVC regime. Controller should only know about service, it does not care and should not know if there are any other layers. Service layer integrates view and other layers: dependent 3rd party services, database repository, files etc. There's whole meat and logic. Repository should only know about service layer. 
Why? Because it makes easier to replace each layer. You want react? sure, it's just one layer. You don't like react but want angular? sure, it's only one layer.You want to change hibernate to jooq? only one contract is to be changed. Preferably designed with interfaces. you are messing view with repository. This mess will get bigger and bigger. That's why MVC concept emerged. 
Seen sql queries from JSP files. It's nothing good, though some seniors programmers that I worked with miss those times. The emotional thrill when application is throwing 500 and you are quickly fixing it with quick SQL query from JSP page.